test/Microsoft.AspNet.Mvc.Core.Test/ModelBinding/MutableObjectModelBinderTest.cs

```diff
@@ -762,8 +762,7 @@ public void GetPropertyValidationInfo_WithIndexerProperties_Succeeds()
             // Arrange
             var bindingContext = new ModelBindingContext
             {
-                // Any type, even an otherwise-simple POCO with an indexer property, would do here.
-                ModelMetadata = GetMetadataForType(typeof(List<Person>)),
+                ModelMetadata = GetMetadataForType(typeof(PersonCollection)),
```

Comment:
This was broken because normally MOMB wouldn't handle `List<Person>`. It was trying to bind the `Capacity` property (among others). The test was really about indexers, so I made a simpler model class.